.github/workflows/deploy-prod-start-argo.yml

```diff
@@ -68,7 +68,7 @@ jobs:
           namespace: ${{ env.OPENSHIFT_TOOLS_NAMESPACE }}
       - name: Display the maintenance page instead of the app
         run: |
-          ./tools/cicd/maintenance/maintenance.sh prod on
+          APPLICATION_NAME=pims-app-prod ./tools/cicd/maintenance/maintenance.sh prod on
```

Comment:
this seems odd - did you actually make this change or perhaps you created your local branch from **test** instead of **dev** ? I ask because I see weird commit history in this PR as well...

<img width="727" height="540" alt="Image" src="https://github.com/user-attachments/assets/d4280ca0-2b26-433d-ab29-26c06b58c67c" />

Reply:
hmmm .... I created this branch from dev :( will try to take out that file from my PR